upgrade/install-1.2.2.php

```diff
@@ -31,6 +33,9 @@
  */
 function upgrade_module_1_2_2($module)
 {
-    return $module->registerHook('actionOrderStatusPostUpdate') && $module->unregisterHook('actionOrderStatusUpdate');
+    $module->registerHook('actionOrderHistoryAddAfter') && $module->unregisterHook('actionOrderStatusUpdate');
+    Configuration::deleteByName(SaferPayConfig::CSS_FILE);
```

Comment:
`SaferPayConfig::CSS_FILE` will not exist as you deleted this value from `SaferPayConfig` class